client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -219,6 +231,7 @@ export default compose( [
 				showClone: 'active' === rewindState.state && ! isAtomic,
 				showDeleteContent: isAtomic || ( ! isJetpack && ! isVip && ! isP2Hub ),
 				showDeleteSite: ( ! isJetpack || isAtomic ) && ! isVip && sitePurchasesLoaded,
+				showHeaderCode: isJetpack || isAtomic,
```

Comment:
Should we show this only to admins? (if the current user has the capability to `manage_options`)

Reply:
Not necessary in this case! The entire page is blocked for non-admins.